ui/src/App.svelte

```diff
@@ -44,7 +44,7 @@
 </script>
 
 <main class:dark={$state.darkmode}>
-  <div class="dark:bg-black dark:text-mlight min-h-screen">
+  <div class="dark:bg-gradient-to-b from-[#181A1C] to-[#0c0c0c] dark:text-mlight min-h-screen">
```

Comment:
Please don't use these custom colours inline, it's super annoying to find and change later. Make a colour entry in `tailwind.config.js` and use the string name.